### PR TITLE
OLSR v1 proto support

### PR DESCRIPTION
**Pull request by leonaard**
Adds support to OLSR v1 routing protocol.
IPv4 and IPv6 version are separated.

### CODE REVIEW

**Comment by ilario** [comment]
Seems working, +1 for merge :D

File: packages/lime-proto-olsr/src/olsr.lua [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	local uci = libuci:cursor()
+	local ipv4 = network.primary_address()
+
+	fs.writefile("/etc/config/olsrd", "")
+
+	uci:set("olsrd", "lime", "olsrd")
+	uci:set("olsrd", "lime", "LinkQualityAlgorithm", "etx_ff")
+	uci:set("olsrd", "lime", "IpVersion", "4")
+
+    uci:set("olsrd", "limejson", "LoadPlugin")
```
Comment (G10h4ck) [inline packages/lime-proto-olsr/src/olsr.lua]: Seems the indentation is not constant, please use tabulator [printf("\t")] as indentation character everywhere

File: packages/lime-proto-olsr/src/olsr.lua [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	uci:set("olsrd", "limehna", "netmask", ipv4:mask():string())
+
+	uci:save("olsrd")
+
+
+end
+
+function olsr.setup_interface(ifname, args)
+	if ifname:match("^wlan%d+_ap") then return end
+	vlanId = args[2] or 23
```
Comment (G10h4ck) [inline packages/lime-proto-olsr/src/olsr.lua]: default vlan-id of protocols should be < 16 see discussion at https://github.com/libre-mesh/lime-packages/pull/10

File: packages/lime-proto-olsr6/src/olsr6.lua [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+	uci:set("olsrd6", "limehna", "prefix", ipv6:prefix())
+
+	uci:save("olsrd6")
+
+
+end
+
+function olsr.setup_interface(ifname, args)
+	if ifname:match("^wlan%d+_ap") then return end
+	vlanId = args[2] or 24
```
Comment (G10h4ck) [inline packages/lime-proto-olsr6/src/olsr6.lua]: default vlan-id of protocols should be < 16 see discussion at https://github.com/libre-mesh/lime-packages/pull/10

**Comment by leonaard** [comment]
I corrected the indentation issues and chenged VLAN ids to the ones Gui suggested in #10

**Pull request merged**